docs/source/getting-started.rst

```diff
@@ -0,0 +1,89 @@
+===============
+Getting Started
+===============
+
+This section provides an introduction on how to get started with using the SymbiFlow toolchain.
+In order to generate a bitstream (or any intermediate file format),
+you can use one of the toolchain tests. Following steps describe the whole
+process:
+
+Clone repository
+----------------
+
+.. code-block:: bash
+
+    git clone --recursive https://github.com/SymbiFlow/symbiflow-arch-defs.git
```

Comment:
The `--recursive` here shouldn't be required, because `make env` should initialize the submodules.

Reply:
Corrected.